src/tools/clippy/tests/ui/unknown_clippy_lints.stderr

```diff
@@ -49,5 +49,16 @@ error: unknown lint: `clippy::const_static_lifetim`
 LL | #[warn(clippy::const_static_lifetim)]
    |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: did you mean: `clippy::redundant_static_lifetimes`
 
-error: aborting due to 8 previous errors
+error: unknown lint: `clippy::missing_docs`
+  --> $DIR/unknown_clippy_lints.rs:17:8
+   |
+LL | #[warn(clippy::missing_docs)]
+   |        ^^^^^^^^^^^^^^^^^^^^
+   |
+help: did you `missing_docs`, a lint with the same name was found in `rustc` lints.
```

Comment:
```suggestion
help: did you mean `missing_docs`; a lint with the same name was found in `rustc` lints
```

or just

```suggestion
help: a lint with the same name was found in `rustc` lints
```

Reply:
I would also reverse this message: `` a lint with the same name exists in `rustc` `` I don't think you need to put the lint name in the `help` message, if you also include it in the `suggestion`.